cluster/gce/util.sh

```diff
@@ -433,6 +433,13 @@ function kube-up {
     --target-tags "${MASTER_TAG}" \
     --allow tcp:443 &
 
+  # We have to make sure the disk is created before creating the master VM, so
+  # run this in the foreground.
+  gcloud compute disks create "${MASTER_NAME}-pd" \
+    --project "${PROJECT}" \
+    --zone "${ZONE}" \
+    --size "10GB"
+
```

Comment:
Yes, it's just cleanup to keep the commands grouped more logically. I'll move it into a separate commit. Is it fine if I leave it in this PR?